src/main/java/org/jitsi/jigasi/xmpp/CallControlMucActivator.java

```diff
@@ -400,6 +401,101 @@ private void updatePresenceStatusForXmppProviders()
         Statistics.updatePresenceStatusForXmppProviders();
     }
 
+    /**
+     * Adds new call control MUC xmpp account using the provided props.
+     *
+     * @param id the id to use for this account(should not contain .).
+     * @param propertiesSet the properties.
+     * @throws OperationFailedException if failed loading account.
+     */
+    public static void addCallControlMucAccount(
+            String id,
+            Set<Map.Entry<String, String>> propertiesSet)
+        throws OperationFailedException
+    {
+        ConfigurationService config
+            = JigasiBundleActivator.getConfigurationService();
+        ProtocolProviderFactory xmppProviderFactory
+            = ProtocolProviderFactory.getProtocolProviderFactory(
+                JigasiBundleActivator.osgiContext,
+                ProtocolNames.JABBER);
+        AccountManager accountManager
+            = ProtocolProviderActivator.getAccountManager();
+
+        // extract the properties, make sure we decode the password
+        HashMap<String, String> properties = new HashMap<>();
+        propertiesSet.stream().forEach(e ->
+        {
+            String property = e.getKey();
+            String value = e.getValue();
+            if (ProtocolProviderFactory.PASSWORD.equals(property))
```

Comment:
Why do we expect that the pasword is Base64 encoded?

Reply:
This is how we configure it in the sip-communicator.properties file, I thought to keep it ... as to be able to use the same template for adding those accounts ... 
And I tested, removing the XMPP account from my config and started adding same properties and values using the REST endpoint ...